modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,93 @@
+
+The `openshift-installer` creates the machine set manifests file with `node-role.kubernetes.io/edge` and `node-role.kubernetes.io/worker` labels applied to each edge worker node that is located in a Local Zone subnet.
+
+.Prerequisites
+
+* You have an access to the OpenShift CLI (`oc`).
+* You deployed your cluster in a Virtual Private Cloud (VPC) with defined Local Zone subnets.
+* You ensured that the machine set for the edge workers on Local Zone subnets specifies the taints for `node-role.kubernetes.io/edge`.
+
+.Procedure
+
+. Create a `deployment` resource YAML file for an example application to be deployed in the edge worker node that operates in a Local Zone subnet. Ensure that you specify the correct tolerations that match the taints for the edge worker node.
++
+.Example of a configured `deployment` resource for an edge worker node that operates in a Local Zone subnet
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment <1>
+metadata:
+  name: <local_zone_name> <2>
+  namespace: <local_workload_zoneName> <3>
+spec:
+  selector: 
+    matchLabels:
+      app: <local_zone_application>
+  replicas: 1
+  template:
+    metadata:
+      labels: 
+        app: <local_zone_application>
+        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME} <4>
+    spec:
+      nodeSelector: <5>
+        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME}
+      tolerations: <6>
+      - key: "node-role.kubernetes.io/edge"
+        operator: "Equal"
+        value: ""
+        effect: "NoSchedule"
+      containers:
+        - image: openshift/origin-node
+          command:
+           - "/bin/socat"
+          args:
+            - TCP4-LISTEN:8080,reuseaddr,fork
+            - EXEC:'/bin/bash -c \"printf \\\"HTTP/1.0 200 OK\r\n\r\n\\\"; sed -e \\\"/^\r/q\\\"\"'
+          imagePullPolicy: Always
+          name: echoserver
+          ports:
+            - containerPort: 8080
+----
+<1> `kind`: Defines the `deployment` resource.
+<2> `name`: Specifies the name of the service. 
+<3> `namespace:` Defines the namespace that the `service` resource targets.
+<4> `machine.openshift.io/zone-group`: Defines the group to where a zone belongs. For example, `us-east-1-nyc-1`.
+<5> `nodeSelector`: Targets edge worker nodes that match the specified labels.
+<6> `tolerations`: Sets the tag where the scheduler can apply a taint to affected nodes. The toleration must match the taint that you defined in the machine set manifest file. 
+
+. Create a `service` resource YAML file for the node. This resource exposes a pod from a targeted edge worker node to services that run inside your Local Zone network.
++
+.Example of a configured `service` resource for an edge worker node that operates in a Local Zone subnet
+[source,yaml]
+----
+apiVersion: v1
+kind: Service <1>
+metadata:
+  name:  <local_zone_service_name>
+  namespace: <local_zone_name>
```

Comment:
same comment above, try to follow the same standard of the Deployment for `namespace`. For the name, can be anything considering the `selector` is correct, but for didactic reasons we could (open to hear from your based in other examples from our docs) create the same name of the app, so copying the Deployment suggestion:

```suggestion
  name: <local_zone_applicatio>
  namespace: <local_zone_application_namespace>
```

Reply:
@dfitzmau you missed the namespace that should match with https://github.com/openshift/openshift-docs/pull/57063/files#diff-6684ff2ad223a0c07c100d71fe49840776acf869d0d2873b6df6675abd3a6684R35

Reply:
Thanks for pointing this out. Not sure why this change was not added to the commit.